integration-tests/actor-tests/nft.test.ts

```diff
@@ -16,14 +16,7 @@ actor('NFT claimer', () => {
 
   setupActor(async () => {
     env = await OptimismEnv.new()
-
-    const factory = new ContractFactory(
-      ERC721.abi,
-      ERC721.bytecode,
-      env.l2Wallet
-    )
-    contract = await factory.deploy()
-    await contract.deployed()
+    contract = new Contract(process.env.ERC_721_ADDRESS, ERC721.abi)
```

Comment:
How exactly is the contract meant to be deployed? It now works under the assumption that the contract already exists. Perhaps if the address is passed in via env var then it doesn't deploy it and otherwise it does?

Reply:
I was thinking about this... I thought it'd be cleaner to have the actors run against pre-existing environments rather than deploying fresh each time. This lets us significantly simplify their implementation.